src/lua/CommunityBalanceMod/FortressPvE/TwiliteShade/TechTreeConstants.lua

```diff
@@ -0,0 +1,25 @@
+local function AppendToEnum( tbl, key )
```

Comment:
same again, and should use the EnumUtils.AppendToEnum function instead of reimplementing it if possible